etools-dialog.html

```diff
@@ -30,14 +30,15 @@
         width: 900px;
       }
       paper-dialog .dialog-title {
-        margin: 0 0 15px 0 !important;
+        margin: 0 !important;
         border-bottom: 1px solid var(--etools-dialog-borders-color, #dedede);
         padding: 10px 24px;
         min-height: 26px;
         @apply(--etools-dialog-title);
       }
       paper-dialog-scrollable {
```

Comment:
Use the `--paper-dialog-scrollable` mixin applied to paper-dialog-scrollable and then add a new element specific mixin like `etools-dialog-scrollable` to be used in the app for more customizations.  Avoid using `!important`.